packages/block-editor/src/components/media-replace-flow/style.scss

```diff
@@ -16,7 +16,6 @@
 }
 
 .block-editor-media-flow__url-input {
-	margin-top: $grid-unit-20;
```

Comment:
why do we need the style change?

Reply:
The margin-top rule has been added before to add a space between the url form and the MediaUpload component, this is how it looks like with the margin top applied: 
![Screen Shot 2020-05-13 at 11 22 42 PM](https://user-images.githubusercontent.com/39377174/81871971-c1e35b80-9570-11ea-8513-afa13fae8816.png)

looks inconsistent right? 
so I just removed the margin

Reply:
right, but won't the media upload component still display for users with the correct permissions?

Reply:
@noahtallen Right, I was just checking that out and it looks inconsistent too now: 

![Screen Shot 2020-05-14 at 12 23 57 AM](https://user-images.githubusercontent.com/39377174/81876009-9618a380-9579-11ea-8dcd-885dd692567b.png)

**What do you think would be the best way to solve this?** 
* I suggest moving the margin rule to the `NavigableMenu` component instead of the url form component, but `NavigableMenu` has no custom className, should we add one?

Reply:
> but NavigableMenu has no custom className, should we add one?

yeah let's try that!

Reply:
Done!